projects/keycloak-angular/src/test.ts

```diff
@@ -26,6 +26,6 @@ getTestBed().initTestEnvironment(
   }
 );
 // Then we find all the tests.
-const context = require.context('./', true, /\.spec\.ts$/);
+// const context = require.context('.', true, /\.spec\.ts$/);
 // And load the modules.
-context.keys().map(context);
+// context.keys().map(context);
```

Comment:
Tests weren't running, so I commented these and for some reason they ran again.
I'd love to get some help with this one :V